src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -27,6 +28,8 @@ namespace SerilogWeb.Classic
     /// </summary>
     public class ApplicationLifecycleModule : IHttpModule
     {
+        private const String StopWatchKey = "stopWatchContextKey";
```

Comment:
Can we use the class/property name for this? E.g. `"SerilogWeb.Classic.ApplicationLifecycleModule.StopWatch"` - so there's less sleuthing required when developers encounter this and wonder where it was added.